samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobVideoObjectTrackingSampleTest.java

```diff
@@ -34,7 +34,7 @@
 public class CreateBatchPredictionJobVideoObjectTrackingSampleTest {
 
   private static final String PROJECT = System.getenv("UCAIP_PROJECT_ID");
-  private static final String MODEL_ID = System.getenv("BATCH_PREDICTION_VIDEO_OBJECT_MODEL_ID");
+  private static final String MODEL_ID = "8609932509485989888";

```

Comment:
same as above? any reason for hardcoding?

Reply:
Acknowledged.